runtime/drivers/duckdb/connectors.go

```diff
@@ -21,14 +21,13 @@ func (c *connection) Ingest(ctx context.Context, source *connectors.Source) erro
 	switch source.Connector {
 	case "file":
 		return c.ingestFile(ctx, source)
-	case "s3":
-		return c.ingestS3(ctx, source)
-	case "gcs":
-		return c.ingestGCS(ctx, source)
+		// TODO: figure out the issue with duckdb and perhaps use this instead of downloading
```

Comment:
Never know what the future brings, but if you want, I'm fine with permanently removing those functions – this approach seems cleaner to me.

Reply:
Ya sounds good. The old implementation should always be available through this PR